### DISCUSSION TITLE
changing the green color of camera window

### DISCUSSION THREAD

**Discussion by klutzzykitty** [blakeblackshear/frigate#6198] [General]
Hello ! 

With any version of Frigate, the default color of the camera window is green when there's no camera signal (like when the cameras are turned off, etc). 

for my frigate instance, I thought of changing it. 
so far I could understand that the camera canvas window displays the `latest.jpg` image when there is input from the camera
_img.src = `${apiHost}/api/${name}/latest.jpg?h=${scaledHeight}${searchParams ? `&${searchParams}` : ''}`;_ 

 But couldn't find any info on where the color green is defined when there is no camera input. 
 
 Can you kindly guide me to the part which has the code to control the camera canvas ?

**Comment by blakeblackshear** [comment]
It's not defined anywhere. That's how an empty frame of 0s renders. Anyway, we already replaced it with a static image with an error message in the current release.

**Reply by klutzzykitty** [reply]
@blakeblackshear 
Thanks Blake! didn't know about the rendering part. 

Yeah I saw that in V0.12 it has been addressed. 

If I were to make the same changes in V0.11.1, is it just enough if I add the following lines to `CameraImage.jsx` in V0.11.1  from V0.12?

1)   `const enabled = config ? config.cameras[camera].enabled : 'True';`
```
2)  return (
    <div className="relative w-full" ref={containerRef}>
      {
        (enabled) ?
          <canvas data-testid="cameraimage-canvas" height={scaledHeight} ref={canvasRef} width={scaledWidth} />
          : <div class="text-center pt-6">Camera is disabled in config, no stream or snapshot available!</div>
      }
      {
        (!hasLoaded && enabled) ? (
          <div className="absolute inset-0 flex justify-center" style={`height: ${scaledHeight}px`}>
            <ActivityIndicator />
          </div>
        ) : null
      }
    </div >
  );
}
```
i.e.adding / replacing V0.11.1's CameraImage.jsx with the above lines from V0.12 ? 

Or are there other files that also have to be changed? 


PS: yeah I use V0.12 as well on my other instance. But one of my instances runs V0.11.1 which can't be 
upgraded as I've lost portainer access to it :)

**Reply by blakeblackshear** [reply]
That won't work. The enabled attribute is only available in 0.12. There isn't a good way to do this in 0.11.
